src/ASM/ASMs3D.C

```diff
@@ -1737,7 +1737,8 @@ bool ASMs3D::integrate (Integrand& integrand,
       Matrix   dNdu, Xnod, Jac;
       Matrix3D d2Ndu2, Hess;
       double   dXidu[3];
-      Vec4     X;
+      double   param[3] = { 0.0, 0.0, 0.0 };
```

Comment:
In 3D there is no need to actually initialize the three param values, since all of them will be assigned anyway before use.

Reply:
done.